test/acceptance/user.acceptance.ts

```diff
@@ -43,7 +47,11 @@ describe('User (acceptance)', () => {
     request = supertest(`http://127.0.0.1:${port}`);
   }
 
-  async function givenUser(data: Partial<User>) {
+  async function givenCreatedUser(data: Partial<User>) {
```

Comment:
Methods called "given" are called as part of test setup, this comes from given-when-then convention, see e.g. https://martinfowler.com/bliki/GivenWhenThen.html

The helper `givenUser` is used in test setup to describe the fact that we are expecting an existing user in the database. The fact that we are using `repository.create` is an implementation detail that should not leak in the helper name.

It's also important that if the user has any required properties, then this helper should provide sensible default values for properties that are not part of caller-supplied `data`. This avoid unnecessary churn in test code when new required properties are added to our models over time, as the application grows. That's the reason for creating this helper and not calling`repository.create` directly.

Please revert this change.

Reply:
@bajtos Thank you for the refresher on GivenWhenThen. I see why it should remain `givenUser` as a setup to the test.